### PR TITLE
Fix comments default for openqa-maintenance bot

### PR DESCRIPTION
**Pull request by coolo**
And remove some of the noisy debug output

### CODE REVIEW

File: openqa-maintenance.py [hunk 1]
```diff
@@ -1029,6 +1026,10 @@ def setup_checker(self):
         return bot
 
 if __name__ == "__main__":
+    requests_log = logging.getLogger("requests.packages.urllib3")
+    requests_log.setLevel(logging.WARNING)
+    requests_log.propagate = False
+
```
Comment (lnussel) [inline openqa-maintenance.py]: Hmm, not sure this belongs here. Maybe we need to leverage
https://docs.python.org/2/library/logging.config.html#logging-config-api

Comment (coolo) [inline openqa-maintenance.py]: urllib3 already leverages it - and this is stackoverflow's recommendation how to shut it up.

Comment (lnussel) [inline openqa-maintenance.py]: urllib uses logger but the config file would have to be parsed by the program that has __main__

Comment (coolo) [inline openqa-maintenance.py]: Huh? what program would that be? I'm not following

**Pull request merged**